### PR TITLE
Changelogbanner

### PR DESCRIPTION
**Pull request by urquha**
This pull request needs to be tested with the local government dashboard, also there are too many styles in the styles file. Test using the branch with very similar name or the same in local government dashboard.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -12,10 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",
     "scripts": {
-        "start": "webpack-serve --config ./webpack.serve.config.js --open --port 8000",
+        "start": "webpack-serve --config ./webpack.serve.config.js --open",
+        "start-mac": "export NODE_OPTIONS=--openssl-legacy-provider && webpack-serve --config ./webpack.serve.config.js --open --port 8000",
         "validate-init": "python _validate_init.py",
         "prepublishOnly": "npm run validate-init",
-        "build:js": "export NODE_OPTIONS=--openssl-legacy-provider &&  ./node_modules/webpack/bin/webpack.js --mode production",
+        "build:js": "webpack --mode production",
```
Comment (Jordan-Hall) [inline package.json:19]: You sure you don't need openssl now?

File: src/lib/components/ChangeLogBanner.react.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+     */
+    LinkComponent: PropTypes.oneOfType([
+        PropTypes.string,
+        PropTypes.func,
+        PropTypes.object
+    ]),
+};
+
+ChangeLogBanner.defaultProps = {
+    updates: [],
```
Comment (Jordan-Hall) [inline src/lib/components/ChangeLogBanner.react.js:37]: Wouldn't supporting null be better here

File: package.json [hunk 3]
```diff
@@ -33,7 +34,10 @@
         "@react-aria/overlays": "^3.8.1",
         "@react-stately/combobox": "^3.0.4",
         "dash": "^3.24.21",
+        "moment": "^2.29.4",
```
Comment (Jordan-Hall) [inline package.json:37]: Please avoid moment choice something like date-fn. Moment not really support and is a bloat

File: package.json [hunk 4]
```diff
@@ -33,7 +34,10 @@
         "@react-aria/overlays": "^3.8.1",
         "@react-stately/combobox": "^3.0.4",
         "dash": "^3.24.21",
+        "moment": "^2.29.4",
         "ramda": "^0.26.1",
+        "react-router-dom": "^6.15.0",
+        "styled-components": "^6.0.7",
```
Comment (Jordan-Hall) [inline package.json:40]: Why styled components?

File: src/lib/fragments/ChangeLogBanner.styles.js [hunk 5]
```diff
@@ -0,0 +1,215 @@
...
+        .li`
+            margin-top: 30px;
+            border-top: 1px solid #e1e1e1;
+            
+            &:first-of-type {
+                border-top: none;
+            }
+            
+            & > * {
+                margin-left: 0.5rem;
```
Comment (Jordan-Hall) [inline src/lib/fragments/ChangeLogBanner.styles.js:163]: Use logical properties ie margin-block etc

**Review by Jordan-Hall — CHANGES_REQUESTED** [review]

**Review by tom-clarke-dluhc — APPROVED** [review]

File: package.json [hunk 6]
```diff
@@ -12,10 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
```
Comment (tom-clarke-dluhc) [inline package.json:12]: The package version needs updating

File: package.json [hunk 7]
```diff
@@ -12,10 +12,11 @@
     "homepage": "https://github.com/communitiesuk/uk-gov-dash-components",
     "main": "build/index.js",
```
Comment (tom-clarke-dluhc) [inline package.json:13]: I can't run the branch from the dashboard code. I got "no module named ...imports".

File: package.json (hunk 1 above)
Comment (urquha) [reply package.json:19]: It didn't have the open SSL before, this has been added recently, it doesn't work in the dap with the export command

Comment (Jordan-Hall) [reply package.json:19]: add dotenv package and add it in the .env. This will be an issue depending on version of node

Comment (Jordan-Hall) [reply package.json:19]: Yes you do. Openssl was upgraded In 18+ but one or more packages use the old one by default. I can update the whole repo if required

File: package.json (hunk 7 above)
Comment (Jordan-Hall) [reply package.json:13]: Hi Tom

Some day I may forget about this but I love open source too much. You need to add "type": "module" to package json. Or make it output as .mjs

File: package.json (hunk 1 above)
Comment (urquha) [reply package.json:19]: If you don't mind that would be really helpful

File: package.json (hunk 7 above)
Comment (LukeDallimore) [reply package.json:13]: Have manged to run this locally and is now working

**Pull request merged**